common/src/androidMain/kotlin/com/trikot/sample/viewmodels/base/ViewModelController.kt

```diff
@@ -0,0 +1,4 @@
+package com.trikot.sample.viewmodels.base
+
+@Suppress("NO_ACTUAL_CLASS_MEMBER_FOR_EXPECTED_CLASS")
+actual typealias ViewModelController = androidx.lifecycle.ViewModel
```

Comment:
I am wondering if it's the right choice to have the type `ViewModelController` associated to a platform specific entity that have nothing to do with controlling a view model (just on the naming side of things). In an Ideal world we would find another name for this and rename `BaseViewModelController` -> `ViewModelController`